cli_helpers/_compat.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+"""Python 2/3 compatibility support."""
```

Comment:
If we move it to `__init__.py`, then our internal utilities would clutter up the public namespace -- e.g. `from cli_helpers import *`.

It seems like it's fairly common practice to use underscored modules internally (e.g. Click, Flask, Werkzeug, Django, and CherryPy).

However, it's also common practice to name internal modules without a leading underscore (e.g. SQLAlchemy and Python Prompt Toolkit).

I'm open to the leading or not-leading underscore. Maybe it's more consistent in dbcli's packages to drop the leading underscore?